guides/doc-Planning_for_Project/master.adoc

```diff
@@ -14,29 +14,36 @@ ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
 
-[[part-Architecture]]
-= {Project} Architecture
+[id="Project_Overview_and_Concepts_{context}"]
+= {Project} Overview and Concepts
```

Comment:
I'm also leaning towards splitting Planning_for_Project into two guides. In its current state, the guide mixes conceptual overview information and practical steps to be taken during deployment, which is.. not great. But it will have to wait after everything has been modularized. It will be easier to do the splitting once we have proper assemblies.

Reply:
> Can we add an attribute for this guide? if so, we could use it here. :)

Let's not do that for now and wait until/if we split this into two guides. So no action.